jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -61,9 +61,14 @@ echo '# BEGIN SECTION: setup the osrf/simulation tap'
 brew tap osrf/simulation
 echo '# END SECTION'
 
-if [[ -n "${PULL_REQUEST_URL}" ]]; then
-  echo "# BEGIN SECTION: pulling ${PULL_REQUEST_URL}"
-  brew pull ${PULL_REQUEST_URL}
+if [[ -n "${ghprbSourceBranch}" ]] && \
+   [[ "${ghprbSourceBranch}" =~ matching_branch\/ ]]
```

Comment:
we need to update the regexp for matching branches accordingly to #572. It would be great if we can avoid the pattern duplication.

Reply:
updated in https://github.com/ignition-tooling/release-tools/pull/575/commits/8a9b0c90b0f1f7f42f7a79e6c6c4931ec586dbab

I suppose we could set a variable in `dependencies_archive.sh`, which would work for homebrew and Ubuntu jobs on build.osrfoundation.org, but not on Windows. We could encode the pattern in a file? It seems simpler to just duplicate it for now

Reply:
For multi-platform support that works on Win and UNIX I can not think in a different option than using a file, yes. Would be great to have this kind of support but by now I'm more worried about having the same regexp in two different files which is a call for problems in the future.

Reply:
I'll open an issue for this in order to move forward

Reply:
https://github.com/ignition-tooling/release-tools/issues/579